src/txn/txn.c

```diff
@@ -531,6 +531,25 @@ __wt_txn_config(WT_SESSION_IMPL *session, const char *cfg[])
 	if (cval.val)
 		F_SET(txn, WT_TXN_IGNORE_PREPARE);
 
+	/*
+	 * Check if prepare timestamp and commit timestamp of a prepared
+	 * transaction to be rounded off.
+	 */
+	WT_RET(__wt_config_gets_def(
+	    session, cfg, "roundup_timestamps.prepared", 0, &cval));
+	if (cval.val)
+		F_SET(txn, WT_TXN_ROUND_PREPARE);
```

Comment:
Could you change this to `WT_TXN_TS_ROUND_PREPARED`, and similarly `WT_TXN_TS_ROUND_READ`